src/main/java/com/google/devtools/build/lib/BUILD

```diff
@@ -195,6 +195,7 @@ java_library(
         "//src/main/protobuf:failure_details_java_proto",
         "//third_party:guava",
         "//third_party:jsr305",
+        "@com_google_protobuf//java/core",
```

Comment:
nit: Just curious, why do we need this now but not previously?

Reply:
Without it the compiler complains:
```
src/main/java/com/google/devtools/build/lib/runtime/BlazeCommandResult.java:39: error: [strict] Using type com.google.protobuf.Any from an indirect dependency (TOOL_INFO: "@com_google_protobuf//java/core"). See command below **
  private final ImmutableList<Any> responseExtensions;
                              ^
 ** Please add the following dependencies: 
  @com_google_protobuf//java/core to //src/main/java/com/google/devtools/build/lib:runtime/blaze_command_result 
 ** You can use the following buildozer command: 
buildozer 'add deps @com_google_protobuf//java/core' //src/main/java/com/google/devtools/build/lib:runtime/blaze_command_result 
```

com.google.protobuf.Any class is not provided by `//src/main/protobuf:any_java_proto` in the new version, because it was denylisted:
https://github.com/protocolbuffers/protobuf/blob/v3.19.6/java/core/BUILD#L174-L179
vs
 https://github.com/protocolbuffers/protobuf/blob/v21.7/java/core/BUILD.bazel#L216-L236

It seems the intent was, whenever you need Any, you shoud depend on the core protobuf library.

I removed any_java as it's not needed.

Reply:
Thanks for the explanation!